float64 p_gain
float64 d_gain
float64 hip_p_gain
float64 hip_d_gain
float64 leg_ang
float64 leg_len
float64 hip_ang
